apis/python/src/tiledbsoma/experiment_query/eq_types.py

```diff
@@ -0,0 +1,23 @@
+from typing import Dict, Optional, Sequence
+
+import pyarrow as pa
+from typing_extensions import TypedDict
+
+
+class ExperimentAxisQueryReadArrowResult(TypedDict, total=False):
```

Comment:
Looking at this again: is there a reason this is a TypedDict rather than an attrs class? It seems like that would let you make `_make_anndata` as a member function and also let users access `result.obs`/`result.var`/`result.X` etc. (unless there is a reason that they definitely *want* to do it using dict access patterns.

Reply:
There isn't a good reason - I'll give it a try.  It feels like it may need a later round of refinement to align better with the new fluent method, but it is a move in that direction....